server/data/cycles.ts

```diff
@@ -0,0 +1,137 @@
+import { cycles, users } from '../config/mongoCollections';
+import { ObjectId } from 'mongodb';
+import { checkObjectId } from '../helpers/error';
+
+export default {
+
+    /**
+     * @description Get a cycle by id
+     * @param {string} id Cycle id
+     * @returns {Promise<Object>} Cycle object
+     */
+    readById: async (id: string): Promise<object> => {
+        if(!checkObjectId(id)){
+            throw new Error('Invalid id');
+        }
+
+        const cycleCollection = await cycles();
+        const cycle = await cycleCollection.findOne({_id: new ObjectId(id)});
```

Comment:
Add type here for variable

Reply:
and similarly elsewhere

Reply:
These would be any types, and according to your preference below, leave them type-less.